src/Components/Patient/PatientHome.tsx

```diff
@@ -30,10 +30,11 @@ import { ErrorHelperText } from "../Common/HelperInputFields";
 import Modal from "@material-ui/core/Modal";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import ExpandLessIcon from "@material-ui/icons/ExpandLess";
```

Comment:
We definitely need to get rid of material UI icons too :)

Reply:
Yep! Done. :)